docs/specs/meta-clients/web3wallet/README.md

```diff
@@ -1,3 +1,3 @@
 # Web3Wallet SDK Overview
 
-The Web3Wallet SDK combines functionality of [Sign](../../clients/sign/README.md) and [Auth](../../clients/auth/README.md) into one offering by giving wallets a dedicated SDK that bundles both APIs together and only exposing the relevant methods
+The Web3Wallet SDK simplifies the functionality of [Sign](../../clients/sign/README.md) by giving wallets SDK that bundles both Sign & Wallet Authentication together and only exposing the relevant methods
```

Comment:
We should add methods and events related to wallet authentication to Web3Wallet client API